src/applications/vaos/appointment-list/components/ConfirmedAppointmentDetailsPage/ConfirmedAppointmentDetailsPage.jsx

```diff
@@ -96,16 +92,23 @@ function ConfirmedAppointmentDetailsPage({
     [cancelInfo.showCancelModal, cancelInfo.cancelAppointmentStatus],
   );
 
-  if (appointmentDetailsStatus === FETCH_STATUS.loading) {
+  if (
+    appointmentDetailsStatus === FETCH_STATUS.failed ||
+    (appointmentDetailsStatus === FETCH_STATUS.succeeded && !appointment)
```

Comment:
This is another error case that generally shouldn't happen. We might need to log an error in here to capture it.